src/pages/PlanPage/subRoutes/CurrentOrgPlan/BillingDetails/PaymentCard/CreditCardForm.tsx

```diff
@@ -17,21 +25,21 @@ function CreditCardForm({ closeForm, provider, owner }) {
     owner,
   })
 
-  function submit(e) {
+  function submit(e: any) {
     e.preventDefault()
-    updateCard(elements.getElement(CardElement), {
+    updateCard(elements!.getElement(CardElement)!, {
```

Comment:
just curious, how do we know `elements` is defined?

Reply:
Mmmm I made the assumption that because we wait for the card element to load prior to entering information and this element object is handled after the user enters information and then submits that we shouldn't need to worry about it being null

Related docs https://docs.stripe.com/stripe-js/react#useelements-hook

Reply:
Where do we do this waiting for the card element to load, is it a suspense boundary in the parent component? Just wondering if we should disable form submission until everything is loaded in, as they show in the docs:

```js
    if (!stripe || !elements) {
      // Stripe.js hasn't yet loaded.
      // Make sure to disable form submission until Stripe.js has loaded.
      return;
    }
```

Reply:
I thought about it more from a practical perspective; a user in a vast majority of cases is not going to immediately hit "submit" right after opening the form.

A user will enter their information into the component, which would need to be loaded first, before they would submit. I get why stripe suggests using the guard, but I don't feel like we need it.

If we feel strongly about adding it though I don't mind making the update

Reply:
Feels like a case where it's better to have it and not need it, than not have it and need it. It doesn't look like too much of a change tho?

Reply:
Practically I don't think we need it, and if we don't have it and _need it_ the element on the form won't load in the first place.

Reply:
Anyway, not a big deal for me to add since we feel strongly about it. Updated in [b95a93f](https://github.com/codecov/gazebo/pull/2930/commits/b95a93f8335f601431b4aef0cf03a9248aa2089b)